examples/js/controls/OrbitControls.js

```diff
@@ -363,7 +363,7 @@ THREE.OrbitControls = function ( object, domElement ) {
 				targetDistance *= Math.tan( ( scope.object.fov / 2 ) * Math.PI / 180.0 );
 
 				// we actually don't use screenWidth, since perspective camera is fixed to screen height
```

Comment:
^ This comment seems to be attempting to explain why clientWidth is not used,  s/client/screen/  - 

If so, is this change correct or the comment outdated?

Reply:
Thanks, @aardgoose This PR is not correct.

Reply:
Okay, was not sure about the comment....

Reply:
Maybe worth changing the comment to read clientWidth rather than screenWidth to prevent any future confusion?

Reply:
Good idea!